addons/account/tests/test_account_move_in_invoice.py

```diff
@@ -2025,3 +2025,55 @@ def test_bill_amount_should_be_editable(self):
                                     'debit': 500,
                                     'tax_line_id': self.company_data['default_tax_purchase'].id,
                                 }])
+
+    def test_invoice_sent_to_additional_partner(self):
+        """
+        Make sure that when an invoice is a partner to a partner who is not
+        the invoiced customer, they receive a link containing an access token,
+        allowing them to view the invoice without needing to log in.
+        """
+
+        # Create a simple invoice for the partner
+        invoice = self.env["account.move"].create({
+            "partner_id": self.partner_a.id,
+            "move_type": "out_invoice",
+            "invoice_date": "2023-04-17",
+            "invoice_line_ids": [(0, 0, {
+                "name": "Test Invoice Line",
+                "quantity": 1,
+                "price_unit": 100,
+            })],
+        })
```

Comment:
We should use `init_invoice` instead:
```suggestion
        invoice = self.init_invoice('out_invoice', partner=self.partner_a, invoice_date='2023-04-17', amounts=[100])
```

It's much more easy to read

Reply:
Done, thanks!